pkg/kubelet/volumes.go

```diff
@@ -79,19 +74,15 @@ func (vh *volumeHost) NewWrapperUnmounter(volName string, spec volume.Spec, podU
 		spec.Volume.Name = wrapperVolumeName
 	}
 
+	//FindPluginBySpec will only return a valid plugin if only one matched
+	// or an error if 0 plugins matched or more than 1 plugin matched
```

Comment:
These comments and the others like it below don't belong here. You should add them to the godoc of the function being called (FindPluginBySpec, NewUnmounter, newVolumeMounterFromPlugins, etc)